vignettes/Monoconal_Antibody_Data.Rmd.orig

```diff
@@ -92,13 +91,14 @@ knitr::kable(con$mabGridSummary)
 Or we can use method chaining to call multiple filter methods and browse the grid. Method chaining is unique to R6 objects and related to the pipe. See Hadley Wickham's [Advanced R](https://adv-r.hadley.nz/r6.html) for more info
 
 ```{r eval=FALSE}
+con$resetMabGrid()
 con$
-  filterMabGrid(using = "hxb2_location", value = c("Env", "gp160"))$
-  filterMabGrid(using = "donor_species", value = "llama")$
+  filterMabGrid(using = "hxb2_location", value = "gp160")$
+  filterMabGrid(using = "donor_species", value = "human")$
+  filterMabGrid(using = "isotype",       value = "IgG")$
```

Comment:
Style: remove excess spaces

Reply:
These arguments have been aligned as other examples in the vignettes.